flutter music player UI

Kavindu-Gayantha opened the pull request:
related toi issue  #1 .

Safnaj commented:
Hi @Kavindu-Gayantha, The UI Looks Cool.

I request to do the following changes to merge the PR.

- [ ] Change music_app to First Letter Capital as Music_App
- [ ] The UI in readme file looks very big
- [ ] Also update a Mockup same as in the Main ReadMe file

Safnaj reviewed (changes requested):
Changes Requested

Kavindu-Gayantha commented:
> Hi @Kavindu-Gayantha, The UI Looks Cool.
> 
> I request to do the following changes to merge the PR.
> 
>     * [ ]  Change music_app to First Letter Capital as Music_App
> 
>     * [ ]  The UI in readme file looks very big
> 
>     * [ ]  Also update a Mockup same as in the Main ReadMe file

Ok , I will make changes.

Kavindu-Gayantha commented:
@Safnaj All done. 
* [x]  Change music_app to First Letter Capital as Music_App

* [x]  The UI in readme file looks very big

* [x]  Also update a Mockup same as in the Main ReadMe file

Safnaj commented on music_app/android/app/src/main/AndroidManifest.xml:
```diff
@@ -7,7 +7,7 @@
     <application
         android:name="io.flutter.app.FlutterApplication"
-        android:label="music_app"
+        android:label="Music_App"
```
Also change the Main Project name as Music_App

Safnaj replied:
Also add ur mockup in the main readme file here

[ReadMe Link](https://github.com/Safnaj/Flutter-UI-Samples/blob/master/README.md)

Kavindu-Gayantha replied:
ok. got it

Safnaj commented:
Hi @Kavindu-Gayantha, I have added hactoberfest-accepted label for you. In order to merge ur PR, pls change the project name as first capital letter. Check the below image.

<img width="567" alt="Screenshot 2020-10-23 at 12 43 17 AM" src="https://user-images.githubusercontent.com/37530024/96919009-c2d9cb00-14c8-11eb-9f12-12d875fe2e53.png">

Kavindu-Gayantha commented:
> Hi @Kavindu-Gayantha, I have added **hactoberfest-accepted** label for you. In order to merge ur PR, pls change the project name as first capital letter. Check the below image.
> 
> <img alt="Screenshot 2020-10-23 at 12 43 17 AM" width="567" src="https://user-images.githubusercontent.com/37530024/96919009-c2d9cb00-14c8-11eb-9f12-12d875fe2e53.png">

previously i changed name in android manifest file also. I thought it solved. now changed in the packages. as Music_App

Safnaj commented:
@Kavindu-Gayantha i was telling to change this not the manifest file. As ler the unpublished code of conduct we follow a proper naming convention. Therefore it has to be a capital letter

Kavindu-Gayantha commented:
> @Kavindu-Gayantha i was telling to change this not the manifest file. As ler the unpublished code of conduct we follow a proper naming convention. Therefore it has to be a capital letter

I thought it might have solved like that, I renamed now again.
